src/tests/Microsoft.PowerFx.Core.Tests/ParseTests.cs

```diff
@@ -113,8 +113,8 @@ public void TexlParseStringLiteralsWithEscapableCharacters(string script)
         [InlineData("1 * 2 + 3 * 4")]
         [InlineData("1 * 2 * 3 + 4 * 5")]
         [InlineData("1 * 2 * 3 * 4 * 5")]
-        [InlineData("2 - 1", "2 + -1")]
-        [InlineData("2 - 1 - 2 - 3 - 4", "2 + -1 + -2 + -3 + -4")]
+        [InlineData("2 - 1", "2 - 1")]
```

Comment:
Thanks for reviewing this, @MikeStall. In general, I would not remove tests. In this case, we are comparing an input expression string against its PrettyPrint string.
Since this fix is meant to replace the "add+negate" approach, this particular test case would fail.
Let me know if it makes sense.

Reply:
Good point.  I see you're just changing the expected value. 
It would still be good to add these input cases:

2+ -1
-1